docs/release_notes/v1.10.8.md

```diff
@@ -82,4 +82,53 @@ instead of JSON.
 
 ### Solution
 
-We implemented a custom BSON Marshal function for our Actor Reminder struct so that the MongoDB state store can correctly serialize ActorReminder data.
\ No newline at end of file
+We implemented a custom BSON Marshal function for our Actor Reminder struct so that the MongoDB state store can correctly serialize ActorReminder data.
+
+# Fixed a Goroutine leak in Actor reminders and timers.
```

Comment:
Please see changes proposed from the PR for 1.11

Reply:
Also for the other issues, including the BSON thing, which has a rather negative tone.

Reply:
Will merge after this is addressed